src/gtest/test_joinsplit.cpp

```diff
@@ -257,7 +257,7 @@ TEST(joinsplit, full_api_test)
 
 TEST(joinsplit, note_plaintexts)
 {
-    uint256 a_sk = uint256S("f6da8716682d600f74fc16bd0187faad6a26b4aa4c24d5c055b216d94516847e");
+    uint252 a_sk = uint252(uint256S("f6da8716682d600f74fc16bd0187faad6a26b4aa4c24d5c055b216d94516847e"));
```

Comment:
I'd expect this to throw an exception because the leading byte is 0x7e. See my comment below about whether or not the constructor should silently mask those bits or throw an exception if they're nonzero.